packages/eslint-config-eslint/types/formatting.d.ts

```diff
@@ -0,0 +1,3 @@
+import type { Linter } from "eslint";
+
+export declare const rules: NonNullable<Linter.Config["rules"]>;
```

Comment:
I think this doesn't reflect the intended usage, which is `import eslintConfigESLintFormatting from "eslint-config-eslint/formatting";`:

https://github.com/eslint/eslint/tree/main/packages/eslint-config-eslint#formatting-config

 This module exports a `Linter.Config` object, not named export `rules`.